graphstorm-processing/tests/test_dist_numerical_transformation.py

```diff
@@ -453,3 +458,91 @@ def test_rank_gauss_reshuffling(spark: SparkSession, check_df_schema, epsilon):
         assert_almost_equal(
             [row["rand"]], expected_vals[i, :], decimal=4, err_msg=f"Row {i} is not equal"
         )
+
+
+def test_json_representation(input_df: DataFrame, check_df_schema):
+    """Test that the generated representation is correct"""
+    dist_numerical_transformation = DistNumericalTransformation(
```

Comment:
You should also test the cases when normalizer is "standard", "guass" or an unknown string.